src/main/java/lotto/lotto/Lotto.java

```diff
@@ -0,0 +1,27 @@
+package lotto.lotto;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Lotto {
+    private static final int MAX_COUNT = 6;
+    private final Set<Integer> numbers = new HashSet<>();
```

Comment:
```suggestion
    private final Set<LottoNumber> numbers = new HashSet<>();
```
로또 번호만 담당하는 객체를 추가해보면 어떨까요?

Reply:
LottoNumber 객체를 추가하였습니다! 말씀해주신 생성 제약을 두었습니다.